contracts/upgradeable_contracts/erc20_to_native/ForeignBridgeErcToNative.sol

```diff
@@ -102,4 +116,100 @@ contract ForeignBridgeErcToNative is BasicForeignBridge, ERC20Bridge, OtherSideB
         emit TokensSwapped(saiContract, erc20token(), curBalance);
         boolStorage[storageAddress] = true;
     }
+
+    function saiTopContract() internal pure returns (ISaiTop) {
+        return ISaiTop(0x9b0ccf7C8994E19F39b2B4CF708e0A7DF65fA8a3);
+    }
+
+    function isTokenSwapAllowed(uint256 _ts) public view returns (bool) {
+        uint256 esTs = saiTopContract().caged();
+        if (esTs > 0 && _ts > esTs) {
+            return false;
+        }
+        return true;
+    }
+
+    function halfDuplexErc20token() public pure returns (ERC20) {
+        return ERC20(0x89d24A6b4CcB1B6fAA2625fE562bDD9a23260359);
+    }
+
+    function setMinHDTokenBalance(uint256 _minBalance) external onlyOwner {
```

Comment:
Let's set some initial value (e.g. `10 ether`) as part of the `migrateToMCD` call

Reply:
Added initial value https://github.com/poanetwork/tokenbridge-contracts/pull/319/commits/360dcaaafefa2069f363becae01c3a71bf4a3a8c